### PR TITLE
resolved dependencies

### PR DESCRIPTION
**Pull request by praveendhawan**

### CODE REVIEW

**Comment by siddharth28** [comment]
@priyank-gupta seems fine to me please check.

File: app/controllers/spree/admin/plans_controller.rb [hunk 1]
```diff
@@ -30,9 +30,14 @@ def update
           render :edit
         end
       end
-      
+
       def destroy
-        @plan.restrictive_destroy_with_api
+        if @plan.restrictive_destroy_with_api
+         flash[:success] = "Plan has been deleted."
```
Comment (priyank-gupta) [inline app/controllers/spree/admin/plans_controller.rb]: @praveendhawan Indentation not correct.

Comment (priyank-gupta) [inline app/controllers/spree/admin/plans_controller.rb]: @praveendhawan I think these are js requests hence we should use `flash.now` instead.

File: app/controllers/spree/admin/recurrings_controller.rb [hunk 2]
```diff
@@ -31,7 +31,12 @@ def update
       end
 
       def destroy
-        @recurring.restrictive_destroy
+        if @recurring.restrictive_destroy
+          flash[:success] = "Recurring has been deleted."
```
Comment (priyank-gupta) [inline app/controllers/spree/admin/recurrings_controller.rb]: @praveendhawan I think these are js requests hence we should use `flash.now` instead.

**Pull request merged**